lib/asciidoctor/converter/semantic-html5.rb

```diff
@@ -34,12 +34,40 @@ def convert_paragraph node
     end
   end
 
+  def convert_inline_anchor node
```

Comment:
Taken from the `Html5Converter`.
The implementation is incomplete because it's not the focus of this pull request.